files/build_templates/docker_image_ctl.j2

```diff
@@ -245,6 +245,11 @@ start() {
     echo "Creating new {{docker_container_name}}$DEV container with HWSKU $HWSKU"
     {%- endif %}
 
+    {%- if docker_container_name == "swss" %}
+    # Obtain the vendor name
+    ASIC_VENDOR=`$SONIC_CFGGEN -y /etc/sonic/sonic_version.yml -v asic_type`
```

Comment:
Use the 'sonic_asic_platform' var instead of defining  "ASIC_VENDOR"

Reply:
fixed.

Reply:
I agree with @neethajohn 's suggestion and it should go to master, 202012 and later braches.

Reply:
@qiluo-msft PR to master created #8361